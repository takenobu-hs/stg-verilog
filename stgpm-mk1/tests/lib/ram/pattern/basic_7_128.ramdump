00000000: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000001: 00000000000000000000000033333333
00000002: 000000000000000000000000cccccccc
00000003: 00000000000000000000000055555555
00000004: 000000000000000000000000aaaaaaaa
00000005: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000006: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000007: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000008: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000009: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000000a: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000000b: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000000c: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000000d: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000000e: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000000f: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000010: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000011: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000012: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000013: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000014: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000015: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000016: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000017: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000018: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000019: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000001a: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000001b: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000001c: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000001d: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000001e: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000001f: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000020: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000021: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000022: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000023: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000024: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000025: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000026: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000027: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000028: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000029: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000002a: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000002b: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000002c: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000002d: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000002e: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000002f: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000030: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000031: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000032: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000033: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000034: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000035: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000036: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000037: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000038: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000039: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000003a: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000003b: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000003c: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000003d: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000003e: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000003f: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000040: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000041: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000042: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000043: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000044: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000045: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000046: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000047: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000048: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000049: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000004a: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000004b: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000004c: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000004d: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000004e: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000004f: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000050: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000051: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000052: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000053: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000054: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000055: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000056: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000057: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000058: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000059: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000005a: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000005b: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000005c: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000005d: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000005e: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000005f: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000060: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000061: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000062: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000063: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000064: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000065: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000066: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000067: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000068: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000069: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000006a: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000006b: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000006c: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000006d: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000006e: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000006f: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000070: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000071: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000072: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000073: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000074: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000075: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000076: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000077: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000078: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
00000079: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000007a: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000007b: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000007c: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000007d: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000007e: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
0000007f: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
