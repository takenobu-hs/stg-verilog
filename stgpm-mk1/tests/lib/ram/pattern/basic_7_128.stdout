monitor   :       0: 1 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :      51: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :      96: (adrs) <- (00)
monitor   :      96: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     101: (dout) -> (xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx)
monitor   :     101: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     116: (adrs,din) <- (00, 000000000000000000000000ffffffff)
monitor   :     116: 0 1 1 00 000000000000000000000000ffffffff xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     121: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     126: (adrs) <- (00)
monitor   :     126: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     130: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000ffffffff
read_ram  :     131: (dout) -> (000000000000000000000000ffffffff)
monitor   :     131: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000ffffffff
monitor   :     140: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     146: (adrs,din) <- (00, 00000000000000000000000000000000)
monitor   :     146: 0 1 1 00 00000000000000000000000000000000 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     151: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     156: (adrs) <- (00)
monitor   :     156: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     160: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 00000000000000000000000000000000
read_ram  :     161: (dout) -> (00000000000000000000000000000000)
monitor   :     161: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 00000000000000000000000000000000
monitor   :     170: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     176: (adrs,din) <- (00, 00000000000000000000000012345678)
monitor   :     176: 0 1 1 00 00000000000000000000000012345678 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     181: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     186: (adrs) <- (00)
monitor   :     186: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     190: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 00000000000000000000000012345678
read_ram  :     191: (dout) -> (00000000000000000000000012345678)
monitor   :     191: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 00000000000000000000000012345678
monitor   :     200: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     206: (adrs,din) <- (00, 000000000000000000000000cafecafe)
monitor   :     206: 0 1 1 00 000000000000000000000000cafecafe xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     211: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     216: (adrs) <- (00)
monitor   :     216: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     220: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000cafecafe
read_ram  :     221: (dout) -> (000000000000000000000000cafecafe)
monitor   :     221: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000cafecafe
monitor   :     230: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     236: (adrs,din) <- (00, 000000000000000000000000cafecafe)
monitor   :     236: 0 1 1 00 000000000000000000000000cafecafe xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     241: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     246: (adrs) <- (00)
monitor   :     246: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     250: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000cafecafe
read_ram  :     251: (dout) -> (000000000000000000000000cafecafe)
monitor   :     251: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000cafecafe
monitor   :     260: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     266: (adrs,din) <- (7f, 000000000000000000000000beefbeef)
monitor   :     266: 0 1 1 7f 000000000000000000000000beefbeef xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     271: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     276: (adrs) <- (7f)
monitor   :     276: 0 1 0 7f xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     280: 0 1 0 7f xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000beefbeef
read_ram  :     281: (dout) -> (000000000000000000000000beefbeef)
monitor   :     281: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000beefbeef
monitor   :     290: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     296: (adrs,din) <- (01, 00000000000000000000000033333333)
monitor   :     296: 0 1 1 01 00000000000000000000000033333333 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     301: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     306: (adrs,din) <- (02, 000000000000000000000000cccccccc)
monitor   :     306: 0 1 1 02 000000000000000000000000cccccccc xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     311: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     316: (adrs,din) <- (03, 00000000000000000000000055555555)
monitor   :     316: 0 1 1 03 00000000000000000000000055555555 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     321: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     326: (adrs,din) <- (04, 000000000000000000000000aaaaaaaa)
monitor   :     326: 0 1 1 04 000000000000000000000000aaaaaaaa xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     331: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     336: (adrs) <- (01)
monitor   :     336: 0 1 0 01 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     340: 0 1 0 01 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 00000000000000000000000033333333
read_ram  :     341: (dout) -> (00000000000000000000000033333333)
monitor   :     341: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 00000000000000000000000033333333
read_ram  :     346: (adrs) <- (02)
monitor   :     346: 0 1 0 02 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 00000000000000000000000033333333
monitor   :     350: 0 1 0 02 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000cccccccc
read_ram  :     351: (dout) -> (000000000000000000000000cccccccc)
monitor   :     351: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000cccccccc
read_ram  :     356: (adrs) <- (03)
monitor   :     356: 0 1 0 03 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000cccccccc
monitor   :     360: 0 1 0 03 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 00000000000000000000000055555555
read_ram  :     361: (dout) -> (00000000000000000000000055555555)
monitor   :     361: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 00000000000000000000000055555555
read_ram  :     366: (adrs) <- (04)
monitor   :     366: 0 1 0 04 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 00000000000000000000000055555555
monitor   :     370: 0 1 0 04 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000aaaaaaaa
read_ram  :     371: (dout) -> (000000000000000000000000aaaaaaaa)
monitor   :     371: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000aaaaaaaa
monitor   :     380: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     386: (adrs,din) <- (00, 000000000000000000000000beefcafe)
monitor   :     386: 0 1 1 00 000000000000000000000000beefcafe xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     391: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     396: (adrs) <- (00)
monitor   :     396: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     400: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000beefcafe
read_ram  :     401: (dout) -> (000000000000000000000000beefcafe)
monitor   :     401: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000beefcafe
unk_cs_ram :     406: (adrs,din) <- (00, 00000000000000000000000000000000)
monitor   :     406: 0 x 1 00 00000000000000000000000000000000 000000000000000000000000beefcafe
monitor   :     410: 0 x 1 00 00000000000000000000000000000000 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     411: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     416: (adrs) <- (00)
monitor   :     416: 0 1 0 00 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     421: (dout) -> (xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx)
monitor   :     421: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
write_ram :     436: (adrs,din) <- (7f, 000000000000000000000000cafebeef)
monitor   :     436: 0 1 1 7f 000000000000000000000000cafebeef xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     441: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     446: (adrs) <- (7f)
monitor   :     446: 0 1 0 7f xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     450: 0 1 0 7f xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000cafebeef
read_ram  :     451: (dout) -> (000000000000000000000000cafebeef)
monitor   :     451: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 000000000000000000000000cafebeef
unk_cs_ram :     456: (adrs,din) <- (7f, ffffffffffffffffffffffffffffffff)
monitor   :     456: 0 1 x 7f ffffffffffffffffffffffffffffffff 000000000000000000000000cafebeef
monitor   :     460: 0 1 x 7f ffffffffffffffffffffffffffffffff xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
monitor   :     461: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     466: (adrs) <- (7f)
monitor   :     466: 0 1 0 7f xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
read_ram  :     471: (dout) -> (xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx)
monitor   :     471: 0 0 x xx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
dump_ram  : dump memory
